src/Resources/contao/config/config.php

```diff
@@ -66,3 +66,7 @@
     'yesno'             => 'Terminal42\LeadsBundle\DataTransformer\YesNoTransformer',
     'uuidToFilePath'    => 'Terminal42\LeadsBundle\DataTransformer\UuidToFilePathTransformer',
 );
+
+// Cron jobs
+$GLOBALS['TL_CRON']['daily']['purgeLeads'] = array('Terminal42\LeadsBundle\EventListener\CronjobListener', 'onDaily');
```

Comment:
Should use the event dispatcher.

Reply:
@Toflar event dispatcher? This is the poor-mans cron job, it does not fire events does it? But it should use a service for sure.